Userland/Utilities/bf.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2024, Ninad Sachania <ninad.sachania@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/StringView.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+
+static void execute(StringView const instructions)
+{
+    Array<u8, 30000> m_data;
```

Comment:
It might be useful to make this size configurable by a command line argument.

Reply:
Done.